### PR TITLE
Added Pippen Collab Info and changed the Staff page to about us

### PR DESCRIPTION
**Pull request by StoneMcYT**
## Changed:
- changes `staff and contributors` tab to `about us`
- added marina bay sands to projects
- some docs on team page

### CODE REVIEW

**Comment by StoneMcYT** [comment]
Just realized there are nav bar issues will fix that in a bit

**Comment by StoneMcYT** [comment]
> 
> 
> Just realized there are nav bar issues will fix that in a bit

Should be fixed in  3fa5b5a

**Comment by tintinkung** [comment]
```diff
diff --git a/Teams/index.html b/Teams/index.html
--- Teams/index.html
+++ Teams/index.html
@@ -35,9 +35,9 @@
               <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
           </li>
           <li class="nav-item">
             <a href="../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
-              <i class="fas fa-atlas"></i>&nbsp;Staff and Contributors</a>
+              <i class="fas fa-atlas"></i>&nbsp;About Us</a>
           </li>
 		      <li class="nav-item">
             <a href="index.html" class="nav-link p-3" style="font-size: large;">
               <i class="fas fa-atlas"></i>&nbsp;Teams</a>
diff --git a/Projects/Pippen_CBD/index.html b/Projects/Pippen_CBD/index.html
--- Projects/Pippen_CBD/index.html
+++ Projects/Pippen_CBD/index.html
@@ -34,10 +34,10 @@
             <a href="../../Contact_Us/contact.html" class="nav-link p-3" style="font-size: large;">
               <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
           </li>
           <li class="nav-item">
-            <a href="../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
-              <i class="fas fa-atlas"></i>&nbsp;Leaders and Contributors</a>
+            <a href="../../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
+              <i class="fas fa-atlas"></i>&nbsp;About Us</a>
           </li>
           <li class="nav-item">
             <a href="../../Teams/index.html" class="nav-link p-3" style="font-size: large;">
               <i class="fas fa-atlas"></i>&nbsp;Teams</a>
diff --git a/Projects/Wat_Ratchantdaram/index.html b/Projects/Wat_Ratchantdaram/index.html
--- Projects/Wat_Ratchantdaram/index.html
+++ Projects/Wat_Ratchantdaram/index.html
@@ -35,9 +35,9 @@
               <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
           </li>
           <li class="nav-item">
             <a href="../../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
-              <i class="fas fa-atlas"></i>&nbsp;Staff and Contributors</a>
+              <i class="fas fa-atlas"></i>&nbsp;About Us</a>
           <li class="nav-item">
             <a href="../../Teams/index.html" class="nav-link p-3" style="font-size: large;"><i
                 class="fas fa-atlas"></i>&nbsp;Teams</a>
           </li>
diff --git a/Projects/projects.html b/Projects/projects.html
--- Projects/projects.html
+++ Projects/projects.html
@@ -35,9 +35,9 @@
               <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
           </li>
           <li class="nav-item">
             <a href="../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
-              <i class="fas fa-atlas"></i>&nbsp;Staff and Contributors</a>
+              <i class="fas fa-atlas"></i>&nbsp;About Us</a>
           </li>
 		      <li class="nav-item">
             <a href="../Teams/index.html" class="nav-link p-3" style="font-size: large;">
               <i class="fas fa-atlas"></i>&nbsp;Teams</a>
diff --git a/Projects/Mandalay_Palace/index.html b/Projects/Mandalay_Palace/index.html
--- Projects/Mandalay_Palace/index.html
+++ Projects/Mandalay_Palace/index.html
@@ -35,9 +35,9 @@
               <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
           </li>
           <li class="nav-item">
             <a href="../../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
-              <i class="fas fa-atlas"></i>&nbsp;Staff and Contributors</a>
+              <i class="fas fa-atlas"></i>&nbsp;About Us</a>
           </li>
           <li class="nav-item">
             <a href="../../Teams/index.html" class="nav-link p-3" style="font-size: large;">
               <i class="fas fa-atlas"></i>&nbsp;Teams</a>
diff --git a/Projects/Luym_Building/index.html b/Projects/Luym_Building/index.html
--- Projects/Luym_Building/index.html
+++ Projects/Luym_Building/index.html
@@ -35,9 +35,9 @@
               <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
           </li>
           <li class="nav-item">
             <a href="../../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
-              <i class="fas fa-atlas"></i>&nbsp;Staff and Contributors</a>
+              <i class="fas fa-atlas"></i>&nbsp;About Us</a>
           </li>
           <li class="nav-item">
             <a href="../../Teams/index.html" class="nav-link p-3" style="font-size: large;">
               <i class="fas fa-atlas"></i>&nbsp;Teams</a>
diff --git a/Minecraft/index.html b/Minecraft/index.html
--- Minecraft/index.html
+++ Minecraft/index.html
@@ -35,9 +35,9 @@
                             <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
                     </li>
                     <li class="nav-item">
                         <a href="../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
-                            <i class="fas fa-atlas"></i>&nbsp;Staff and Contributors</a>
+                            <i class="fas fa-atlas"></i>&nbsp;About Us</a>
                     </li>
                     <li class="nav-item">
                         <a href="../Teams/index.html" class="nav-link p-3" style="font-size: large;">
                             <i class="fas fa-atlas"></i>&nbsp;Teams</a>
diff --git a/Contact_Us/contact.html b/Contact_Us/contact.html
--- Contact_Us/contact.html
+++ Contact_Us/contact.html
@@ -34,9 +34,9 @@
               <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
           </li>
           <li class="nav-item">
             <a href="../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
-              <i class="fas fa-atlas"></i>&nbsp;Staff and Contributors</a>
+              <i class="fas fa-atlas"></i>&nbsp;About Us</a>
           </li>
           <li class="nav-item">
             <a href="../Teams/index.html" class="nav-link p-3" style="font-size: large;">
               <i class="fas fa-atlas"></i>&nbsp;Teams</a>
diff --git a/About_Us/index.html b/About_Us/index.html
--- About_Us/index.html
+++ About_Us/index.html
@@ -35,9 +35,9 @@
               <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
           </li>
           <li class="nav-item">
             <a href="./index.html" class="nav-link p-3" style="font-size: large;"><i class="fas fa-atlas">
-            </i>&nbsp;Leader and Contributors</a>
+            </i>&nbsp;About Us</a>
           </li>
 		      <li class="nav-item">
             <a href="../Teams/index.html" class="nav-link p-3" style="font-size: large;">
               <i class="fas fa-atlas"></i>&nbsp;Teams</a>

```

File: index.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
           </li>
           <li class="nav-item">
             <a href="./Staff_and_Contributors/index.html" class="nav-link p-3" style="font-size: large;">
```
Comment (tintinkung) [inline index.html:55]: you forgets 1 mor thing

**Comment by tintinkung** [comment]
iz dis completed

**Review by tintinkung — APPROVED** [review]
## ✔️ Approved
ok all fix is completed, ready to merge

**Pull request merged**